<APP_NAME>/__init__.py

```
@@ -0,0 +1,2 @@
+default_app_config = "<APP_NAME>.apps.AppMonitorConfig"
```

Ruff (0.12.2)

Invalid module name: '<APP_NAME>'

(N999)